Frameworks/AddressBook/_ABAddressBook.mm

```diff
@@ -0,0 +1,52 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "_ABAddressBook.h"
+#import "UWP/WindowsApplicationModelContacts.h"
+
+@interface _ABAddressBook ()
+
+@end
+
+@implementation _ABAddressBook
```

Comment:
It was set-up as a class with future operations in mind (like adding/updating/removing contacts) -- an _ABAddressBook will need to maintain extra state and have more functionality to interact with the contact store and/or individual contacts as future features are implemented, particularly the save operation.

---
In reply to: [73073364](https://github.com/Microsoft/WinObjC/pull/786#discussion_r73073364) [](ancestors = 73073364)